test/codegen/table_scan_translator_test.cpp

```diff
@@ -35,6 +36,122 @@ class TableScanTranslatorTest : public PelotonCodeGenTest {
     CreateAndLoadAllColsTable();
   }
 
+  void ExecuteTileGroupTest(peloton::LayoutType layout_type) {
```

Comment:
Can you modify the base test class functions to accept a `LayoutType`? Then you should reuse the create/loading functions rather than creating a one-off function. This also allows other tests to use custom table layouts for free.

Reply:
Correct, the tests do not use the inserter, but instead directly insert into the table [here](https://github.com/cmu-db/peloton/blob/master/test/codegen/testing_codegen_util.cpp#L134).

Reply:
@pmenon I have modified the `CreateTestTables` function to get the layout as a parameter. However, since its called from the constructor, so the entire test suit would get called with the given layout. I have however moved the `CreateAndLoad` function to the base class. Please let me know if you want me to change anything else.